task-03-codewars-and-numpy/15-maria-lytvyn/Masha.py.py

```diff
@@ -0,0 +1,36 @@
+#A point on a plane is denoted as (x,y). The origin of the plane is denoted as (0,0) Given a list of points, select the n points that are closest to the origin.
+#The given list is never empty.
+
+import numpy as np
+
+a = np.array([], tuple)
+b = tuple()
+print("Enter the  5 elements of list: ")
+lenght = []
+for i in range(5):
+    x = float(input("Enter x: "))
+    y = float(input("Enter y: "))
+    c = ((x**2) + (y**2))**0.5
+    lenght.append(c)
+    b = (x,y)
+    a = np.append(a, b)
+print("**********The array of points: ***************************")
+print (a)
+print("**********The array of lenghts of vectors: ***************************")
+print (lenght)
+min = lenght[0]
```

Comment:
На мою думку, частина коду від 21 до 36 рядка коду може бути винесена в окрему функцію або в цикл (так як двічі повторюється одна і та сама ділянка коду, за виключенням 29 рядка, який можна викликати між викликами функцій окремо або залишити в кінці цикла, що не змінить результата роботи програми)
`
```suggestion
 #виклик функції перший раз
 lenght.pop(index)
 #виклик функції  другий раз
````